extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/AbstractDataSourceProducer.java

```diff
@@ -65,6 +71,10 @@
     @Inject
     public TransactionSynchronizationRegistry transactionSynchronizationRegistry;
 
+    @Inject
+    @Any
+    public Instance<AgroalPoolInterceptor> agroalPoolInterceptors;
```

Comment:
Is there a good reason to have this field `public` instead of `protected` or `package-private`? And the same applies to other injection points in this class.

Reply:
No idea why. Changed all the injection points to `package-private`